Add Music Licensing article and update copyright policy (#2783)
ppy/osu-wiki: wiki/Legal/Copyright/Music_Licensing/en.md

> # Music Licensing

Reviewer: ```suggestion
---
legal: true
---

# Music Licensing
```

> The osu! team regularly reaches out to artists that the community loves and new artists whose music is appropriate for use within the community and game. Once an agreement is established, artists are listed under the [Featured Artists](/wiki/Featured_Artists) program.

Reviewer: > Once an agreement is established

an agreement to what? the first sentence only mentions who you agree with, not what you agree to (might be obvious to most readers, but language is important on these kinds of pages)

the "what" (licensing music) should be your first sentence

Artists interested in joining the Featured Artist program are encouraged to contact osu! for licensing negotiations.

> ## Terms

Reviewer: missing info about usage of songs off-platform, like in livestreams or replay videos. whether it's covered or not, I think that's important to disclose here

Author: covered by line 11?

Reviewer 2: i've addressed this with a refactor on line 11. i think @cl8n was more talking about non-official uploads off-platform, which we intentionally do not cover in our licensing.

Reviewer: yep

> Music licensed by osu! is freely available for download from the [Featured Artists listing](https://osu.ppy.sh/beatmaps/artists) in `.osz` files, which are beatmap templates prepared with timing data. A one minute automatically generated preview of each track is available for streaming on its artist listing. Because songs are used as a catalog for community members to create beatmaps, there is no guarantee that they will be used in the game.

Reviewer: not sure if mentioning beatmaps/timing is a good idea for this page if it's meant to be read by ppl who may not know anything about osu

Reviewer 3: I think it might have to be used in lamens terms since this could be read by either a record company or an artist who has not heard of the game before

> Music licensed by osu! is freely available for download from the [Featured Artists listing](https://osu.ppy.sh/beatmaps/artists) in `.osz` files, which are beatmap templates prepared with timing data. A one minute automatically generated preview of each track is available for streaming on its artist listing. Because songs are used as a catalog for community members to create beatmaps, there is no guarantee that they will be used in the game.

Reviewer: last sentence reads kinda pessimistic. "some of them may not be immediately used as in-game content" for the last part?

Author: it's true though, some songs likely won't be used. better to keep it truthful

Author: jk i changed it

> We look to cover usage within the game (and related website), which does not include further usage on other streaming platforms (ie. user uploads to Youtube). One exception is that we utilise Featured Artist tracks as background music during official events, such as [osu! World Cup](wiki/Tournaments#-osu!-world-cup) live-streams. An exclusion clause can be negoatiated on request to completely restrict usage to within the game only.

Reviewer: ```suggestion
We look to cover usage within the game and related website, which does not include further usage on other streaming platforms (e.g. user uploads to YouTube). One exception is that we utilise Featured Artist tracks as background music during official events, such as [osu! World Cup](/wiki/Tournaments#-osu!-world-cup) live-streams. An exclusion clause can be negoatiated on request to completely restrict usage to within the game only.
```

> osu! licences all music under non-exclusive terms, allowing simultaneous licensing with other platforms (as it is our belief that we should not be limiting good music to only our platform). We generally prefer to negotiate licences on perpetual terms due to the nature of usage (users expect their creations to stay up indefinitely). Payment is provided as a fixed negotiable once-off royalty.

Reviewer: ```suggestion
osu! licences all music under non-exclusive terms, allowing simultaneous licensing with other platforms (it is our belief that we should not be limiting good music to only our platform). We generally prefer to negotiate licences on perpetual terms due to the nature of usage (users expect their creations to stay available indefinitely). Payment is provided as a fixed negotiable once-off royalty.
```

## Contact

> **Email**: [music@ppy.sh](mailto:music@ppy.sh)

Reviewer: ```suggestion
**Email:** [music@ppy.sh](mailto:music@ppy.sh)
```
just looks a little better /shrug

> If you are interested in negotiating licensing of your work within osu!, please contact the email address above with samples of your work. We will decide if your music is a good fit for osu! and elaborate on the terms of the license.

Reviewer 4: ```suggestion
If you are interested in negotiating licensing of your work within osu!, please contact us using the email address above with samples of your work. We will decide if your music is a good fit for osu! and elaborate on the terms of the license.
```

---
legal: true
---

# Music Licensing

Artists who enter a licensing agreement with osu! are listed under the [Featured Artists](/wiki/Featured_Artists) program. The osu! team regularly reaches out to artists that the community loves and new artists whose music is appropriate for use within the community and game.

Artists interested in joining the Featured Artist program are encouraged to contact osu! for licensing negotiations.

## Terms

> Music licensed by osu! is freely available for download from the [Featured Artists listing](https://osu.ppy.sh/beatmaps/artists) in `.osz` files, which are level (beatmap) templates. A one minute automatically generated preview of each track is available for streaming on its artist listing. Because songs are used as a catalog for community members to create beatmaps, some of them may not be immediately used as in-game content.

Reviewer: ```suggestion
Music licensed by osu! is freely available for download from the [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists) in `.osz` files, which are level (beatmap) templates. A one-minute automatically generated preview of each track is available for streaming on its artist's page. Because songs are used as a catalog for community members to create beatmaps, some of them may not be immediately used as in-game content.
```

> We look to cover usage within the game and related website, which does not include further usage on other streaming platforms (e.g. user uploads to YouTube). One exception is that we utilise Featured Artist tracks as background music during official events, such as [osu! World Cup](/wiki/Tournaments#-osu!-world-cup) live-streams. An exclusion clause can be negoatiated on request to completely restrict usage to within the game only.

Reviewer 5: ```suggestion
We look to cover usage within the game and related website, which does not include further usage on other streaming platforms (e.g. user uploads to YouTube). One exception is that we utilise Featured Artist tracks as background music during official events, such as [osu! World Cup](/wiki/Tournaments#-osu!-world-cup) live-streams. An exclusion clause can be negotiated on request to completely restrict usage to within the game only.
```

Reviewer: I thought this was fixed cuz I saw "negotiated" written correctly somewhere else in the article
rip
